activemodel/lib/active_model/errors.rb

```diff
@@ -5,7 +5,6 @@
 require "active_support/core_ext/object/deep_dup"
 require "active_model/error"
 require "active_model/nested_error"
-require "forwardable"
```

Comment:
shouldn't we require `active_support/core_ext/module/delegation.rb` instead here? @skipkayhil

Reply:
> is already required in all frameworks through `active_support/rails`.